### PR TITLE
Update storage.rst

### PR DESCRIPTION
**Pull request by jls2933**

### CODE REVIEW

File: cookbook/profiler/storage.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Switching the Profiler Storage
 ==============================
 
-By default the profile stores the collected data in files in the cache directory.
+By default the profile stores the collected data in files in the cache directory, under ``%kernel.cache_dir%/profiler``.
```
Comment (javiereguiluz) [inline cookbook/profiler/storage.rst]: I agree to add the explicit location, but what about rewording it a bit:

```
By default the profile stores the collected data in files in the ``%kernel.cache_dir%/profiler/`` directory.
```

Comment (xabbuh) [inline cookbook/profiler/storage.rst]: I like this proposal.

Comment (WouterJ) [inline cookbook/profiler/storage.rst]: :+1: for @javiereguiluz' proposal.

@jls2933 can you please edit your PR to use this rewording? If you don't have the time or you don't know how to do it, feel free to ask for help or we can do it after merging the PR.

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
I like it - thanks @jls2933!

**Pull request closed without merging**